etherpad.yml

```diff
@@ -5,6 +5,9 @@ services:
     etherpad:
         image: jitsi/etherpad
         restart: ${RESTART_POLICY}
+        environment:
```

Comment:
@sapkra This clashe with your desire to drop the etherpad container, and this is pretty much the reason why I wanted it to stay, in case we wanted to customize the config, and it looks like the day has come. WDYT?

Reply:
I think a better solution would be to just make all values configuration via env vars in the official image. The code for it is already there but was not applied to all values. I will create a PR for it to change this in the etherpad repository.

Reply:
https://github.com/ether/etherpad-lite/pull/3873

Reply:
Nice! Thanks for the support in etherpad